addons/repair/tests/test_repair.py

```diff
@@ -777,3 +777,27 @@ def test_trigger_orderpoint_from_repair(self):
         self.assertFalse(move.repair_id)
         self.assertEqual(move.location_id, self.stock_warehouse.lot_stock_id)
         self.assertEqual(move.location_dest_id, self.stock_location_14)
+
+    def test_copy_repair_product_with_different_groups(self):
+        """
+            This test checks if the product can be copied with users that don't have access on the Inventory app
+        """
+        product_templ = self.env['product.template'].create({
+            'name': 'Repair Consumable',
+            'type': 'consu',
+            'create_repair': True,
+        })
+        mitchell_user = self.env['res.users'].create({
+            'name': 'Mitchell not Admin',
+            'login': 'm_user',
+            'email': 'm@user.com',
+            'groups_id': [(6, 0, self.env.ref('base.group_system').ids)],
+        })
+        self.env.cache.remove(product_templ, product_templ._fields['create_repair'])
```

Comment:
```suggestion
        product_templ.invalidate_recordset(['create_repair'])
```
This is the more idiomatic way of removing a record's field from the cache.

Reply:
Thanks for the feedback, updated :)